baseplate/diagnostics/tracing/publisher.py

```diff
@@ -3,12 +3,12 @@
 
 import requests
 
-from . import MAX_SPAN_SIZE, MAX_QUEUE_SIZE
 from baseplate import config, metrics_client_from_config
 from baseplate._compat import configparser
+from baseplate._compat import urlparse
+from baseplate.diagnostics.tracing import MAX_SPAN_SIZE, MAX_QUEUE_SIZE
 from baseplate.message_queue import MessageQueue, TimedOutError
 from baseplate.retry import RetryPolicy
-from baseplate._compat import urlparse
 from baseplate._utils import BatchFull, RawJSONBatch, TimeLimitedBatch
```

Comment:
why not use the same style of imports in this file as the rest of baseplate?

Reply:
I'd like to move to absolute imports only. Do you prefer the relative ones?

Reply:
I do not!  I prefer absolute, but I also like consistency 😅 

If we want to move to absolute, this might be a good time to do that but it's also fine to kick that can.

Reply:
I solemnly swear to not kick that can too far down the road but I don't want this PR to balloon too much or the merge conflicts will get insane the longer it rots.